upload/MultiFile.php

```diff
@@ -1,77 +1,83 @@
 <?php
+
 namespace upload;
 
 class MultiFile
 {
-  public $Files = array();
-  public $Errors = array();
+    public $Files = [];
+    public $Errors = [];
 
-  function __construct($key)
-  {
-    for ($i=0; $i < count($_FILES[$key]['name']); $i++) {
-      if (is_uploaded_file($_FILES[$key]['tmp_name'][$i])) {
-        $data = array(
-          'name' => $_FILES[$key]['name'][$i],
-          'tmp_name' => $_FILES[$key]['tmp_name'][$i]
-          );
-        $this->Files[] = new File($key, $data);
-      }
+    public function __construct($key)
+    {
+        for ($i = 0; $i < count($_FILES[$key]['name']); $i++) {
+            if (is_uploaded_file($_FILES[$key]['tmp_name'][$i])) {
+                $data = [
+          'name'     => $_FILES[$key]['name'][$i],
+          'tmp_name' => $_FILES[$key]['tmp_name'][$i],
+          ];
+                $this->Files[] = new File($key, $data);
+            }
+        }
     }
-  }
 
-  public function Upload($newPath)
-  {
-    $returns = array();
-    if (empty($this->getErrors())) {
-      foreach ($this->Files as $File ) {
-        $returns[] = $File->Upload($newPath);
-      }
-      return $returns;
-    } else {
-      return false;
+    public function Upload($newPath)
```

Comment:
__[phpmd]__
* The method Upload is not named in camelCase.
* The variable $File is not named in camelCase.

<!-- Comment by [SideCI](https://sideci.com) -->